include/dds/pub/detail/CoherentSet.hpp

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2019, Proyectos y Sistemas de Mantenimiento SL (eProsima).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#ifndef EPROSIMA_DDS_PUB_DETAIL_COHERENT_SET_HPP_
+#define EPROSIMA_DDS_PUB_DETAIL_COHERENT_SET_HPP_
+
+//#include <dds/pub/detail/TCoherentSetImpl.hpp>
+//#include <org/opensplice/pub/CoherentSetDelegate.hpp>
+
+/**
+ * @cond
+ * Ignore this file in the API
+ */
+
+namespace dds {
+namespace pub {
+namespace detail {
+
+//typedef dds::pub::TCoherentSet<org::opensplice::pub::CoherentSetDelegate> CoherentSet;
+class CoherentSet
+{
+};
```

Comment:
If the class is to be replaced by the typedef in the future, add a tracking TODO.